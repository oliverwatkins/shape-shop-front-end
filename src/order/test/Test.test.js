import React from 'react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
	const tree = renderer
		.create(<div page="http://www.facebook.com">Facebook2</div>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});