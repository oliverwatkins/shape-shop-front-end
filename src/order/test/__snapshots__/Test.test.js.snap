// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  page="http://www.facebook.com"
>
  Facebook2
</div>
`;
